src/modelgauge/default_annotator_set.py

```diff
@@ -1,22 +1,16 @@
-from typing import Any, Dict
+from typing import Any, Dict, List
 
-from modelgauge.annotator import Annotator
 from modelgauge.annotator_set import AnnotatorSet
-from modelgauge.annotators.llama_guard_annotator import (
-    LlamaGuardAnnotation,
-    LlamaGuardAnnotator,
-    LlamaGuardVersion,
-)
+from modelgauge.annotators.llama_guard_annotator import LlamaGuardAnnotation
 from modelgauge.auth.together_key import TogetherApiKey
-from modelgauge.single_turn_prompt_response import (
-    TestItemAnnotations,
-)
+from modelgauge.secret_values import InjectSecret
+from modelgauge.single_turn_prompt_response import TestItemAnnotations
 
 
 class DefaultAnnotatorSet(AnnotatorSet):
 
     ANNOTATOR_NAME = "llama_guard_2"
-    annotators: dict[str, Annotator] = {}
+    annotators: List[str] = ["llama_guard_2"]
```

Comment:
Did you perhaps miss a bit of duplication between these two lines?

Reply:
Ah good point. Fixed